Packages/manifest.json

```diff
@@ -2,12 +2,19 @@
   "dependencies": {
     "com.unity.2d.sprite": "1.0.0",
     "com.unity.2d.tilemap": "1.0.0",
+    "com.unity.ads": "3.7.5",
```

Comment:
Revert this. We don't need these packages.

Reply:
The entire json, or just that line?

Reply:
Keep the JSON, but revert all the changes made to the file in this PR.

Reply:
Done!